WORKSPACE

```diff
@@ -37,12 +37,12 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/istio/envoy/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
-# Commit date: 7/28/20 - Branch: release-1.7
-ENVOY_SHA = "ab94b5e8ff075e2f64a0e68e5bbc42f53a07b182"
+# Commit date: 2020-08-02 - Branch: release-1.7
+ENVOY_SHA = "68efe3d7de8421af0261c63b53e08ae95936fb88"
 
-ENVOY_SHA256 = "f2f536247bc1ebffded45bc3063f56b30919cb04b00ccb584c81b3b741fc5038"
+ENVOY_SHA256 = "b86a2f5f0c34d1f44b981ad3571257519bdfa4127cbdcb11f4ef18a7ba53cf99"
 
-ENVOY_ORG = "istio"
+ENVOY_ORG = "PiotrSikora"
```

Comment:
NOTE: This needs to be updated once https://github.com/istio/envoy/pull/248 and https://github.com/istio/envoy/pull/249 are merged.